### PR TITLE
fix(inji-522): enable the location service before initiating the vc sharing.

### PR DESCRIPTION
**Pull request by vijay151096**

### CODE REVIEW

File: machines/bleShare/scan/scanMachine.ts [hunk 1]
```diff
@@ -679,8 +680,21 @@ export const scanMachine =
           },
         },
         checkingLocationService: {
-          initial: 'checkingPermissionStatus',
+          initial: 'checkLocationService',
```
Comment (KiruthikaJeyashankar) [inline machines/bleShare/scan/scanMachine.ts:683]: Can we rename the parent state name / child state name accordingly as we have similar name for both states

Comment (vijay151096) [reply machines/bleShare/scan/scanMachine.ts:683]: parent state is named as check**ing**LocationService as it is continuing operation, we named the child as checkLocationService, as it is a operation that is performed currently.

which gives more clarity, pls let me know if you have any other concerns.

File: android/app/build.gradle [hunk 2]
```diff
@@ -104,7 +102,7 @@ android {
 
     namespace 'io.mosip.residentapp'
 
-    
+
     namespace 'io.mosip.residentapp'
```
Comment (swatigoel) [inline android/app/build.gradle:106]: Delete this duplicate entry for namespace

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: android/app/build.gradle (hunk 2 above)
Comment (vijay151096) [reply android/app/build.gradle]: removed.

File: machines/bleShare/scan/scanMachine.ts (hunk 1 above)
Comment (KiruthikaJeyashankar) [reply machines/bleShare/scan/scanMachine.ts:683]: The state checkLocationService checks if location is turned on or not, where as checkingLocationService is combination of states for check if location turned on -> if yes check if app has permission to access location -> if yes go ahead or request.
Considering this , the parent is performing a more functionality than just checking for location service. Should be bring in more generic state name for the parent considering its functionalities? (like checkLocationStatus)

**Review by swatigoel — APPROVED** [review]

File: machines/bleShare/scan/scanMachine.ts (hunk 1 above)
Comment (vijay151096) [reply machines/bleShare/scan/scanMachine.ts:683]: updated it @KiruthikaJeyashankar

**Review by KiruthikaJeyashankar — APPROVED** [review]

**Pull request merged**